1-js/01-getting-started/3-code-editors/article.md

```diff
@@ -1,46 +1,46 @@
 
-There are two main types of code editors: IDEs and lightweight editors. Many people use one tool of each type.
+कोड संपादकों के दो मुख्य प्रकार हैं: आईडीई और हल्के संपादक। बहुत से लोग प्रत्येक प्रकार के एक उपकरण का उपयोग करते हैं।
 
-## IDE
+## आईडीई
 
-The term [IDE](https://en.wikipedia.org/wiki/Integrated_development_environment) (Integrated Development Environment) refers to a powerful editor with many features that usually operates on a "whole project." As the name suggests, it's not just an editor, but a full-scale "development environment."
+[आईडीई] (https://en.wikipedia.org/wiki/Integrated_development_environment) (इंटीग्रेटेड डेवलपमेंट एनवायरनमेंट) शब्द एक शक्तिशाली संपादक को कई विशेषताओं के साथ संदर्भित करता है जो आमतौर पर "संपूर्ण परियोजना" पर काम करता है। जैसा कि नाम से पता चलता है, यह सिर्फ एक संपादक नहीं है, बल्कि एक पूर्ण पैमाने पर "विकास का माहौल" है।
 
-An IDE loads the project (which can be many files), allows navigation between files, provides autocompletion based on the whole project (not just the open file), and integrates with a version management system (like [git](https://git-scm.com/)), a testing environment, and other "project-level" stuff.
+एक आईडीई परियोजना को लोड करता है (जो कई फाइलें हो सकती हैं), फाइलों के बीच नेविगेशन की अनुमति देता है, पूरे प्रोजेक्ट के आधार पर स्वत: पूर्णता प्रदान करता है (न केवल खुली फाइल), और एक संस्करण प्रबंधन प्रणाली (जैसे [git] (https: //) के साथ एकीकृत करता है git-scm.com/)), एक परीक्षण वातावरण, और अन्य "प्रोजेक्ट-स्तर" सामान।
 
-If you haven't selected an IDE yet, consider the following options:
+यदि आपने अभी तक आईडीई का चयन नहीं किया है, तो निम्नलिखित विकल्पों पर विचार करें:
 
-- [Visual Studio Code](https://code.visualstudio.com/) (cross-platform, free).
-- [WebStorm](http://www.jetbrains.com/webstorm/) (cross-platform, paid).
+- [विजुअल स्टूडियो कोड] (https://code.visualstudio.com/) (क्रॉस-प्लेटफॉर्म, मुफ्त)।
+- [WebStorm] (http://www.jetbrains.com/webstorm/) (क्रॉस-प्लेटफ़ॉर्म, भुगतान किया गया)।
 
-For Windows, there's also "Visual Studio", not to be confused with "Visual Studio Code". "Visual Studio" is a paid and mighty Windows-only editor, well-suited for the .NET platform. It's also good at JavaScript. There's also a free version [Visual Studio Community](https://www.visualstudio.com/vs/community/).
+विंडोज के लिए, "विज़ुअल स्टूडियो" भी है, "विज़ुअल स्टूडियो कोड" के साथ भ्रमित होने की नहीं। "विज़ुअल स्टूडियो" एक सशुल्क और शक्तिशाली विंडोज-केवल संपादक है, जो .NET प्लेटफॉर्म के लिए अच्छी तरह से अनुकूल है। यह जावास्क्रिप्ट में भी अच्छा है। एक मुक्त संस्करण [विजुअल स्टूडियो कम्युनिटी] (https://www.visualstudio.com/vs/community/) भी है।
 
-Many IDEs are paid, but have a trial period. Their cost is usually negligible compared to a qualified developer's salary, so just choose the best one for you.
+कई आईडीई का भुगतान किया जाता है, लेकिन परीक्षण अवधि होती है। उनकी लागत आमतौर पर एक योग्य डेवलपर के वेतन की तुलना में नगण्य है, इसलिए बस आपके लिए सबसे अच्छा एक का चयन करें।
```

Comment:
आपके लिए सबसे अच्छा एक का चयन करें। this can be अपने आप के लिए सबसे अच्छा एक का चयन करें।